generated/nixnetsocket/nixnetsocket_service.cpp

```diff
@@ -55,12 +55,22 @@ namespace nixnetsocket_grpc {
     try {
       auto socket_grpc_session = request->socket();
       nxSOCKET socket = session_repository_->access_session(socket_grpc_session.id(), socket_grpc_session.name());
+
       auto addr = allocate_output_storage<nxsockaddr, SockAddr>();
       nxsocklen_t addrlen {};
-      auto status = library_->Accept(socket, &addr, &addrlen);
+      auto init_lambda = [&] () {
+        auto socket_out = library_->Accept(socket, &addr, &addrlen);
```

Comment:
Annoying subtle change. When you `Accept`, you pass in a socket (the listening socket) and get out a new socket. So `Accept` needs to be an `init_method`.

I also wanted to use `socket` as the name of both the input and the output. That "just works" in grpc because of the request/response split, but we need a different `cpp_local_name` for one or the other in the implementation. So there are a few codegen changes for that.